app/Config/bootstrap.php

```diff
@@ -103,12 +103,12 @@
  */
 App::uses('CakeLog', 'Log');
 CakeLog::config('debug', array(
-	'engine' => 'File',
-	'types' => array('notice', 'info', 'debug'),
-	'file' => 'debug',
+    'engine' => 'File',
```

Comment:
__[staging][code_sniffer]__
* Array key not aligned correctly; expected 26 spaces but found 4